delfin/common/config.py

```diff
@@ -96,6 +96,23 @@
 CONF.register_opts(global_opts)
 
 
+southbound_security_opts = [
+    cfg.BoolOpt('enable_ssl',
```

Comment:
If this is enabled. it will be applicable for all south bound drivers ? what if one driver wants ssl_disbaled? .
I suggest pass ssl_enabled as an input in access_info .